tez-tests/pom.xml

```diff
@@ -129,6 +129,23 @@
       <artifactId>bcprov-jdk15on</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-api-mockito2</artifactId>
+      <version>2.0.2</version>
```

Comment:
not a tez committer, but I'd expect that dependency version to be set in the base pom.xml and referenced twice

Reply:
Addressed your concerns. Thanks for the suggestion @steveloughran